src/main/java/com/sampleweb/pages/PCVideoGamesPage.java

```diff
@@ -0,0 +1,86 @@
+package com.sampleweb.pages;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class PCVideoGamesPage extends HomePage {
+
+  public static String PATH = "https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dvideogames&field-keywords=";
+  private final String PAGE_TITLE = "//h1[text()='PC and Video Games']";
```

Comment:
This could be made local in the isLoaded() method as it's not used anywhere else.